addons/test_mail_full/tests/test_mail_thread_phone.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.test_mail_full.tests.common import TestMailFullCommon, TestMailFullRecipients
+from odoo.tests import tagged, users
+
+
+@tagged('mail_thread')
+class TestMailThreadPhone(TestMailFullCommon, TestMailFullRecipients):
+    """ Test mail.thread.phone mixin, its tools and API """
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestMailThreadPhone, cls).setUpClass()
+        cls.test_phone_records, cls.test_phone_partners = cls._create_records_for_batch(
+            'mail.test.sms.bl',
+            5
+        )
+        cls.test_phone_records += cls.env['mail.test.sms.bl'].create([
+            {
+                'phone_nbr': '+32475110505',
+                'mobile_nbr': '+32475000505',
+            }, {
+                'phone_nbr': '0032475110606',
+                'mobile_nbr': '0032475000606',
+            }
```

Comment:
:disguised_face: 
```suggestion
            },
```

(same for other method arguments in this file)